### DISCUSSION TITLE
Can we insert a dashboard in a website project?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#9114] [Q&A]
### Description

I'd like to ask if dashboard is meant to be developed by itself, i.e. it cannot be inserted like a page in a website project. 

In my [test website project](https://github.com/hopezh/oatNotes), I inserted the gapminder.qmd in the navbar, it renders like the following locally: 

![2024-03-17_10-15-33](https://github.com/quarto-dev/quarto-cli/assets/23435810/bd087c75-6277-41f4-99f6-54120613e699)

...and it's not rendered when published on [github pages](https://hopezh.github.io/oatNotes/about.html).

**Comment by jimjam-slam** [comment]
It seems like the page [_is_ rendered on GitHub Pages](https://hopezh.github.io/oatNotes/dashboard), but the link to it is missing from the other pages' primary navigation.

**EDIT:** actually, the dashboard link appears on [some pages](https://hopezh.github.io/oatNotes/expand_codeblock_width.html) but [not others](https://hopezh.github.io/oatNotes/quarto_tabset.html)!

**Reply by hopezh** [reply]
Thanks for check it out. So, is this regarded as an issue that **dashboard appears only on some pages**?

**Comment by mcanouil** [comment]
FYI
- https://github.com/quarto-dev/quarto-cli/discussions/8262


<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/772bc6b3-6785-4c9b-baf3-adf4db0d1a30">

Source: https://github.com/mcanouil/quarto-issues-experiments/tree/main/quarto-cli-9114

**Reply by hopezh** [reply]
Yes, a `dashboard` can be added as one section in the `navbar`.

```yml
project:
  type: website
  output-dir: docs

website:
  title: "OatNotes"
  navbar:
    left:
      - href: index.qmd
        text: Home
      - about.qmd
      - dashboard.qmd

  sidebar:
```

My [test repo](https://hopezh.github.io/oatNotes/dashboard.html) updated:

![2024-04-25 22 01 56](https://github.com/quarto-dev/quarto-cli/assets/23435810/caffcf8e-d2a0-4e6a-a794-b6b0ea972e4a)
